_books/ion-1-1/src/binary/e_expressions.md

```diff
@@ -120,326 +136,583 @@ EF 01
+| `()`                  | _&lt;No variadics, no AEB>_ |
+| `(a b c)`             | _&lt;No variadics, no AEB>_ |
+| `(a b c?)`            | `------cc`                  |
+| `(a b* c?)`           | `----ccbb`                  |
+| `(a+ b* c?)`          | `--ccbbaa`                  |
+| `(a+ b c?)`           | `----ccaa`                  |
+| `(a+ b* c? d*)`       | `ddccbbaa`                  |
+| `(a+ b* c? d* e)`     | `ddccbbaa`                  |
+| `(a+ b* c? d* e f?)`  | `ddccbbaa ------ff`         |
+| `(a+ b* c? d* e+ f?)` | `ddccbbaa ----ffee`         |
+
+Each pair of bits in the AEB indicates what kind of expression to expect in the corresponding argument position.
+
+| Bit sequence | Meaning                                                                                                                  | `?` | `*` | `+` |
+|:------------:|:-------------------------------------------------------------------------------------------------------------------------|:---:|:---:|:---:|
+|     `00`     | An **empty stream**. No bytes are present in the corresponding argument position.                                        |  ✅  |  ✅  |  ❌  |
+|     `01`     | A **single expression** of the declared encoding is present in the corresponding argument position.                      |  ✅  |  ✅  |  ✅  |
+|     `10`     | A **[expression group](#expression-groups)** of the declared encoding is present in the corresponding argument position. |  ❌  |  ✅  |  ✅  |
+|     `11`     | _Reserved_. A bitmap entry with this bit sequence is illegal in Ion 1.1.                                                 |  ❌  |  ❌  |  ❌  |
+
+As noted in the table above:
+* An empty stream (`00`) cannot be used to encode an argument for a parameter with a cardinality of `one-or-more`.
+* An expression group (`10`) cannot be used to encode an argument for a parameter with a cardinality of `zero-or-one`.
+
+#### Expression groups
+
+This section describes the encoding of an expression group. For an explanation of what an expression group is and how to use it,
+see _[Expression groups](../todo.md)_.
+
+An expression group begins with a [`FlexUInt`](primitives/flex_uint.md). If the `FlexUInt`'s value
+is:
+* **greater than zero**, then it represents the number of bytes used to encode the rest of the expression group. The reader
+  should continue reading expressions of the declared encoding until that number of bytes has been consumed.
+* **zero**, then it indicates that this is a delimited expression group and the processing varies according to
+  whether the declared encoding is tagged or tagless. If the encoding is:
+  * **[tagged](#tagged-encoding)**, then each expression in the group begins with an opcode. The reader
+    must consume tagged expressions until it encounters a terminating `END` opcode (`0xF0`).
+  * **[tagless](#tagless-encodings)**, then each expression in the group has no leading opcode; there is no
```

Comment:
I suggest rewording this slightly to emphasize that the first byte(s) that follows the expression group's starting FlexUInt 0 byte is a `FlexUInt` length prefix. I think talking about the expressions first risks confusion.